tests/utils/pod/pod.go

```diff
@@ -25,7 +25,16 @@ func DefinePod(podName string, namespace string, image string) *corev1.Pod {
 
 // RedefinePodWithLabel adds label to given pod manifest.
 func RedefinePodWithLabel(pod *corev1.Pod, label map[string]string) {
-	pod.ObjectMeta.Labels = label
+	newMap := make(map[string]string)
```

Comment:
A couple of comments about your changes in this function:
1. Be aware that you're changing the implementation in a non-backwards-compatible way: This function was replacing all the pod labels with the function's parameter ones. Now, you're adding parameter's ones to the pod's existing ones. This might affect other test cases. Even if no other TC is affected by this change, I recommend, at least, to add this change to the PR description. Another option is just creating a new function for appending labels to pod's ones, like `pod.AppendLabels(pod *corev1.Pod, labels map[string]string)`.

2. Why do you need to create a new map to add the labels to the existing pod's map? I believe this code does the same:
```
func RedefinePodWithLabel(pod *corev1.Pod, labels map[string]string) {
  for name, value := range labels {
    pod.Labels[name] = value
  }
}
```